client/client.go

```diff
@@ -749,10 +748,7 @@ tsoBatchLoop:
 		}
 		// Start to collect the TSO requests.
 		maxBatchWaitInterval := c.option.getMaxTSOBatchWaitInterval()
-		if err = tbc.fetchPendingRequests(dispatcherCtx, maxBatchWaitInterval); err != nil {
-			log.Error("[pd] fetch pending tso requests error", zap.String("dc-location", dc), errs.ZapError(errs.ErrClientGetTSO, err))
-			return
-		}
+		tbc.fetchPendingRequests(dispatcherCtx, maxBatchWaitInterval)
```

Comment:
When the context is done, the original implementation will return directly.

Reply:
Fixed, PTAL.